spikingjelly/activation_based/op_counter/spikesim/formulas.py

```
@@ -0,0 +1,71 @@
+    patch_control = (
+        float(metadata["out_channel_tiles"])
+        * float(stats["active_patch_tile_count"])
+        * config.patch_control_energy_pj
+    )
+    active_row_count_by_tile = stats.get("active_row_count_by_tile")
+    input_tile_channels = metadata.get("input_tile_channels")
+    if active_row_count_by_tile is not None and input_tile_channels is not None:
+        xbar_rows = 0.0
+        for row_count, tile_channels in zip(
+            active_row_count_by_tile, input_tile_channels
+        ):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)